### DISCUSSION TITLE
How to change LED offset per key?

### DISCUSSION THREAD

**Discussion by Friheber** [onlaj/Piano-LED-Visualizer#438] [Q&A]
My Keystation for some reason has an octave offset I can't seem to get fixed. How can I change the code so each LED is one button? What I mean is you reduce the stripe to 88 LED and solder them to 1 button each, the Problem is I need to reduce the gap between the LEDs which is nearly impossible to solder because I need to cut them so short that there won't be any soldering points be left. So my only other way around is to reduce the LEDs in between with coding.

Many thanks in advance

**Comment by onlaj** [comment]
Hey,
there is a "low density" mode that works with 60 LEDs per meter strips. Does that help or do you need something else?
